src/libsync/networkjobs.cpp

```diff
@@ -328,21 +337,21 @@ bool LsColJob::finished()
         connect( &parser, SIGNAL(directoryListingIterated(const QString&, const QMap<QString,QString>&)),
                  this, SIGNAL(directoryListingIterated(const QString&, const QMap<QString,QString>&)) );
         connect( &parser, SIGNAL(finishedWithError(QNetworkReply *)),
-                 this, SIGNAL(finishedWithError(QNetworkReply *)) );
+                 this, SIGNAL(parserFinishedWithError(QNetworkReply *)) );
```

Comment:
Actually, looking at the code, it seems that this signal is unused anyway, is it not?  So maybe just remove it.